charts/apisix/values.yaml

```diff
@@ -211,12 +211,21 @@ etcd:
   service:
     port: 2379
 
-  replicaCount: 3
+  statefulset:
+    replicaCount: 3
 
+  persistence:
+    enabled: true
+    size: 8Gi
+    # storageClass: "-"
 
 dashboard:
   enabled: false
-
+  config:
```

Comment:
If we add field in values, Please show us where it will be used.
I think we can not add dashboard configuration here, but should add it into `chars/apsix-dashboard/values.yaml`.
cc @tao12345666333 Am I right ?

Reply:
@gxthrj Yeah, I'm very aware of this DRY(dont-repeat-yourself) stuff, that's why I explained in the description above that a minimized config for `dashboard` is necessary due to the different helm release name would cause a potential issue, which the service name of `etcd` would have another name that would make `dashboard` failing to connect, however, I still partially agree with you as it is a nice-to-have stuff

Reply:
We currently have two main usage scenarios

* `helm install apisix apisix/apisix --set dashboard.enabled=true`  install apisix and apisix-dashboard.
* `helm install dashboard apisix/apisix-dashboard` only install apisix-dashboard.

In fact, here we use the subchart method for quoting, just make sure that the configuration here is consistent with the configuration in subchart.

But as we hope, we hope to reduce repeat works.
Let me have a quick try and give you an answer later

Reply:
Sure thing, let's do what you advised